frontend/app/src/electron-main/http.ts

```diff
@@ -119,7 +119,8 @@ function handleRequests(req: IncomingMessage, res: ServerResponse, cb: Callback)
       return;
     }
   }
-  const basePath = checkIfDevelopment() ? path.join(__dirname, '..', 'public') : __dirname;
+  const dirname = path.dirname(new URL(import.meta.url).pathname);
```

Comment:
we are on node > 20.11 so we should be able to use `import.meta.dirname`

https://stackoverflow.com/a/50052194